var/www/skitter.fanfic.dev/index.html

```diff
@@ -1,88 +1,111 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang=en>
 	<head>
 		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<title>skitter.fanfic.dev</title>
-		<style type="text/css">
-			body { max-width: 960px; }
-			body { font-size: 1.2em; }
-			body h2 { font-size: 1.5em; }
-			.u { font-weight: bold; }
-			.c { font-family: monospace; }
-		</style>
```

Comment:
Is there a benefit to splitting this into an extra file? I suppose as long as the server/client supports http2 then splitting it out doesn't hurt, am I remembering that right?

Reply:
Any specific benefit? No. None comes to mind. I just like to separate my structure from my styling. Yes, you're right about http2. 
Also HTML tags `<strong>` and `<b>` automatically have `font-weight: bold;` applied to them.
HTML tag `code` has `font-family: monospace;` applied to it.

Reply:
I think splitting it out makes more sense when it's large, you want to control caching behavior separately, or you've given up and plan on requiring a bunch of requests to the server (that _might_ be coalesced) to render your page.

I'm a bit hesitant to split this out and add a requirement to hit google's servers for what is such a small README. Do you happen to know if there's any public statistics on average font availability without requiring extra requests? I know there's the minimal web safe font list https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Fundamentals#web_safe_fonts . I tried skimming https://caniuse.com , but that seems to just be whether the property itself is usable and not whether a particular value is.

Maybe this is too "old man yelling at cloud", but choosing to include "Pro" in the name of what _appears_ to be a free font was a questionable decision :p It looks like https://docs.rs vendors Source Code Pro, and viewing their source it looks like they correctly include the license: https://github.com/rust-lang/docs.rs/blob/master/static/SourceCodePro-LICENSE.md . So that's possibly an option.

Ultimately my preference here is to leave this as a single self-contained file, that possibly lists extra fonts in the css for people who have them installed already. A more involved change that statically builds the html with syntax highlighting and uses vendored font files might be acceptable as a separate PR, but I think there's lower hanging fruit like an openapi spec or similar.

Reply:
- I think splitting it out makes more sense when it's large, you want to control caching behavior separately, or you've given up and plan on requiring a bunch of requests to the server (that might be coalesced) to render your page.
  - I guess you're right it's only a couple of lines anyways. Done.
- I'm a bit hesitant to split this out and add a requirement to ...
  - Changed it to use **Verdana** for the body and **Courier (New)** for code.

Reply:
Thanks!